tests-integration/tests/grpc.rs

```diff
@@ -30,6 +30,7 @@ use servers::http::prometheus::{
     PrometheusResponse,
 };
 use servers::server::Server;
+use tests_integration::database::Database;
```

Comment:
In this file, https://github.com/GreptimeTeam/greptimedb/blob/e4101925607726481962000486b32c926d09a0f1/src/servers/tests/grpc/mod.rs#L129-L137 need `Database` client, but it calls several function that is `not public`, so I keep it here for further discussion.